docs/contributing/contributing.rst

```diff
@@ -186,6 +186,16 @@ This is described in the next section.
 Making Your Own Toil Docker Image
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+**Note!**  Toil only supports checking if quay.io and docker.io images exist.  To use 
+an image not hosted on docker.io or quay.io, you will have to specifically override this 
+by specifying ``TOIL_APPLIANCE_SELF`` with the ``[override]`` modifier (though be warned, 
+Toil may loop forever when launching a cluster if this docker image does not, in fact, 
```

Comment:
It will no longer loop forever, but will fail to launch.

Reply:
Alright, I can modify this.